### ISSUE TITLE
👍

### ISSUE THREAD

**Issue by yingzhuo** [subchen/go-cli#1]
这个牛逼了，我今天刚看见。 写个 kubectl 那么多子命令的CLI应该也不难了。

**Comment by subchen** [comment]
Welcome for use in your application. Kindly share if you have any comments.

**Comment by yingzhuo** [comment]
* 假设我不使用`git`，而使用`svn`。 希望您不强迫我使用`app.BuildInfo`。
    * ```
	app.BuildInfo = &cli.BuildInfo{
		Timestamp:   "",
		GitBranch:   "",
		GitCommit:   "",
		GitRevCount: "",
	}
       ```
* 我希望通过中文呈现`Help`页面，好像没办法优雅做到。
    * `cli.HelpTemplate = "替换成中文模板"`

**Comment by yingzhuo** [comment]
cli.Context再加一个`GetDuration() time.Duration`方法就好了。

我想象一下啊。

```
my-app --timeout=100ms
my-app --timeout=60s
my-app --timeout=1h
my-app --timeout="2010-10-24 00:00:00"
```

**Comment by yingzhuo** [comment]
```
my-app --int 0xFF  
my-app --int 0660
my-app --int64 0xFFFF
my-app --int64 0660
```

不知道是否支持非十进制数的配置?

**Comment by subchen** [comment]
1. If you use SVN, you have to rewrite version using `app.ShowVersion = ...`. Do you have better practice？ 

2. Currently, `cli.Context` is only support build-in type, you can use `Flag.Value = &...` for more type.

    Following usage can be enhanced to support all types used in `Flag.Value`.
    ```go
    var timout time.Duration;
    context.Get("name", &timeout);
    ```

3. `0`, `0b10101`, `0xFF`, `077` are supported. More details see `strconv.ParseInt(s, 0, 0)`

**Comment by yingzhuo** [comment]
Help 模板里，我希望有一个 `SEE ALSO` section。 我看到man文档里这里一般放一点 doc-url 什么的。

**Comment by subchen** [comment]
Good point

**Comment by yingzhuo** [comment]
Thank for merging (#2) . release v2.0.1 ? please.

**Comment by subchen** [comment]
Thanks for your pull request.
Released as `v2.0.1`
